extension/latextext.py

```diff
@@ -95,9 +96,7 @@ class SvgTransformer:
 
     # matrix multiplication helper function
     def _matmult(self, a, b):
-        zip_b = zip(*b)
-        # uncomment next line if python 3 :
-        # zip_b = list(zip_b)
+        zip_b = list(zip(*b)) if PY3 else zip(*b)
```

Comment:
It's a redundant use of `list(...)` but this works just fine on Python 2, no need to make it conditional.